code/modules/mining/mine_turfs.dm

```diff
@@ -338,7 +341,13 @@ turf/unsimulated/mineral/ChangeTurf(var/turf/N, var/tell_universe=1, var/force_l
 	if (mineral && mineral.result_amount)
 		for (var/i = 1 to mineral.result_amount - mined_ore)
 			DropMineral()
-
+	switch(rockernaut)
+		if(TRUE)
```

Comment:
epic

Reply:
big if true

Reply:
no really, stop trolling with the defines